protocol/x/bridge/module_test.go

```diff
@@ -78,16 +78,6 @@ func TestAppModuleBasic_RegisterCodec(t *testing.T) {
 	var buf bytes.Buffer
 	err := cdc.Amino.PrintTypes(&buf)
 	require.NoError(t, err)
```

Comment:
For these tests, should we validate that `buf.String()` is empty or something? Not sure if `buf.String()` has other stuff

Reply:
it isn't empty, it is formatted like this when it is empty
```
| Type | Name | Prefix | Length | Notes |
| ---- | ---- | ------ | ----- | ------ |
```